homeassistant/components/fritz/common.py

```diff
@@ -217,21 +217,21 @@ def update(self, dev_info, dev_home, consider_home):
         """Update device info."""
         utc_point_in_time = dt_util.utcnow()
 
+        consider_home_evaluated = (
```

Comment:
Let's not use in-line if-else statements that span multiple lines, it's hard to follow what's going on. Write it as a normal if-else

Reply:
@chemelli74 🙂
Will change it back to a normal if-else